docs/editor_guide.md

```diff
@@ -85,7 +85,6 @@ they should remove the label for the preceding stage of the review process.
 ## Workflow
 
 ### Initial submission
-
 Once the editor confirmed that the lesson was suitable to be sent out to review,
 these steps need to be taken:
 1. add review badge to the README for the repository using the code below and replacing `<issue number>` with the issue associated with the submission:
```

Comment:
does the editor do this or ask the author to?

Reply:
In at least one case, I opened a PR to add the badge to the README, but I have also shared the Markdown snippet for the badge in the review thread and asked the authors to add it themselves. I recommend that you take the latter approach, and will update the guidance here accordingly.